seshdash/views.py

```diff
@@ -691,3 +713,87 @@ def historical_data(request):
         context_dict['sort_keys'] = sort_data_dict.keys()
         context_dict['sort_dict'] = sort_data_dict
         return render(request, 'seshdash/historical-data.html', context_dict);
+
+@login_required
+def time_series_graph(request):
+    context_dict = {}
+    if request.method == 'POST':
+        client=Influx()
+        measurement=request.POST.get('measurement','')
+        time=request.POST.get('time','')
+        measurement_units=BoM_Data_Point.SI_UNITS
+        units=measurement_units[measurement]
+        active_id = request.POST.get('active_id','')
+        active_site=Sesh_Site.objects.filter(id=active_id)
+        active_site_name=active_site[0].site_name
+        time_delta_dict = {'24h':{'hours':24},'7d':{'days':7},'30d':{'days':30}}
+        time_delta = time_delta_dict[time]
+        time_bucket_dict = {'24h':'30m','7d':'12h','30d':'1d'}
+        time_bucket=time_bucket_dict[time]
+        time_series_values=client.get_measurement_bucket(measurement,time_bucket,'site_name',active_site_name,time_delta)
+
+        graph_values = []
+
+        for values in time_series_values:
+            graph_values.append([values['time'],values['mean']])
+
+        for unformatted_values in graph_values:
+            unformatted_values[0]=get_epoch_from_datetime(datetime.datetime.strptime(unformatted_values[0],"%Y-%m-%dT%H:%M:%SZ"))
+        print graph_values
```

Comment:
please comment  out or remove print statements. You can use debug logging where appropriate.